web/themes/custom/atrium/templates/media/media--jcc-form--full.html.twig

```diff
@@ -6,6 +6,16 @@
 #}
 <article{{ attributes.addClass(classes) }}>
   {{ title_suffix.contextual_links }}
+
+  {% include '@molecules/blocks/hero/hero.twig' with {
+    hero: {
+      title: media.label ~ ' (' ~ media.field_number.value ~ ')',
```

Comment:
I was wrong and "media" is the key that you need not "elements". "media" is an object and a Drupal entity so you can apply methods to it. I think I'd have to demo to explain fully.